src/app/admin/invite/invite-form.tsx

```diff
@@ -0,0 +1,99 @@
+'use client'
+
+import { TextInput, PasswordInput, Button, Input, Checkbox, Text, Flex } from '@mantine/core'
+import { useMutation } from '@tanstack/react-query'
+import { useForm } from '@mantine/form'
+import { notifications } from '@mantine/notifications'
+import { OrganizationSelect } from './organization-select'
+import Link from 'next/link'
+import { adminInviteUserAction } from './admin-users.actions'
+import { InviteUserFormValues, inviteUserSchema, zodResolver } from './admin-users.schema'
+import { randomString } from '@/lib/string'
+import { reportError } from '@/components/errors'
+
+export function InviteForm() {
+    const studyProposalForm = useForm({
+        mode: 'uncontrolled',
+        validate: zodResolver(inviteUserSchema),
+        validateInputOnBlur: true,
+        initialValues: {
+            firstName: '',
+            lastName: '',
+            email: '',
+            organizationId: '',
+            password: randomString(8),
+            isReviewer: false,
+            isResearcher: false,
+        } as InviteUserFormValues,
```

Comment:
we shouldn't need this `as InviteUserFormValues` right?

Reply:
we kinda do, putting it here also types the useForm so that the it's typed everywhere.  However it'd be clearer to be explicit and use `useForm<InviteUserFormValues>`, I'll update to do that

Reply:
yeah i generally try to avoid `as` because it's bitten me in the past pretending like it actually complies when it doesnt if you add or remove a prop later on 😭